model/gen-ai/registry.yaml

```diff
@@ -90,6 +90,16 @@ groups:
         type: double
         brief: The presence penalty setting for the GenAI request.
         examples: [0.1]
+      - id: gen_ai.request.encoding_formats
+        stability: experimental
+        type: string[]
+        brief: The encoding formats requested in an embeddings operation, if specified.
+        examples:
+          - ['base64']
+          - ['float', 'binary']
+        note: >
+          In some APIs the encoding formats are called embedding types.
```

Comment:
Maybe these `APIs` should be `GenAI systems` to match the attribute?

Reply:
Thanks. commit 2350ab0